src/sentry/eventstore/base.py

```diff
@@ -198,6 +200,28 @@ def get_prev_event_id(self, event, filter):
         """
         raise NotImplementedError
 
+    def get_oldest_event_id(self, event, filter):
+        """
+        Gets the earliest event given a current event and some conditions/filters.
+        Returns a tuple of (project_id, event_id)
+
+        Arguments:
+        event (Event): Event object
+        filter (Filter): Filter
+        """
+        raise NotImplementedError
+
+    def get_latest_event_id(self, event, filter):
```

Comment:
nit: naming is a bit inconsistent: 
oldest/newest 
earliest/latest

Reply:
Thanks, updated